Sources/FoundationEssentials/Platform.swift

```diff
@@ -220,6 +246,15 @@ extension Platform {
         // For Linux, read /proc/self/exe
         return try? FileManager.default.destinationOfSymbolicLink(
             atPath: "/proc/self/exe").standardizingPath
+#elseif os(Windows)
+        let hFile = GetModuleHandleW(nil)
+        let dwLength: DWORD = GetFinalPathNameByHandleW(hFile, nil, 0, FILE_NAME_NORMALIZED)
+        return withUnsafeTemporaryAllocation(of: WCHAR.self, capacity: Int(dwLength)) { lpBuffer in
+            guard GetFinalPathNameByHandleW(hFile, lpBuffer.baseAddress, dwLength, FILE_NAME_NORMALIZED) == dwLength - 1 else {
+                return nil
+            }
+            return String(decodingCString: lpBuffer.baseAddress!, as: UTF16.self)
```

Comment:
The other platforms call `.standardizingPath` on the result, should we do the same here for consistency?

Reply:
Technically that is already done - `GetFinalPathNameByHandleW` will resolve the symlink and standardize the path in the process. We get the fully normalized location that the file is at.

Reply:
Ah I see, ok if it's already fully standardized the same way that `standardizePath` works then that seems fine to leave it as-is here. Could we add a comment to note that just so it's clear why it's different than the other OS code paths?

Reply:
Sure! That sounds reasonable.